sdk/core/azure-core/src/main/java/com/azure/android/core/util/DateTimeRfc1123.java

```diff
@@ -14,6 +19,8 @@
  * during serialization and deserialization.
  */
 public final class DateTimeRfc1123 {
+    private static final ClientLogger LOGGER = new ClientLogger(DateTimeRfc1123.class);
```

Comment:
I recall an old conversation with @anuchandy about avoiding using static loggers in Android but I can't really recall the actual reason. I'll ask him separately and come back to check if we'd prefer to go with something else.